client/js/lounge.js

```diff
@@ -844,7 +844,7 @@ $(function() {
 
 		socket.emit("input", {
 			target: chat.data("id"),
-			text: "/whois " + name
+			text: "/whois " + name + " " + name
```

Comment:
Ah, sorry. I had a squiz, it's not really defined anywhere but servers out there process it without an issue. Some like Insp also seem to require it to send all the WHOIS info. Should be fine, yeah.